OasisDEX/oasis-borrow
# [L2] Aave V3 Multiply L2 product cards

# [Product Cards for L2](https://app.shortcut.com/oazo-apps/story/8478/product-cards-for-l2)

  
## Changes 👷‍♀️
- added necessary configs to the `@oasisdex/addresses` packages
- added configs for the L2 product cards
  
## How to test 🧪
- toggle `OptimismAaveV3`
- toggle `ArbitrumAaveV3`
- see the Aave V3 Optimism and Arbitrum cards on Homepage, Multiple and Assets page
